<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileView</class>
 <widget class="QWidget" name="ProfileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Профиль пользователя</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_user">
          <property name="toolTip">
           <string>Обязательное поле. Не более 150 символов. Только буквы, цифры и символы @/./+/-/_.</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_phone">
          <property name="toolTip">
           <string>Введите ваш номер телефона</string>
          </property>
          <property name="inputMask">
           <string notr="true">+9 (999) 999-99-99; </string>
          </property>
          <property name="text">
           <string notr="true">+ () --</string>
          </property>
          <property name="cursorPosition">
           <number>18</number>
          </property>
          <property name="placeholderText">
           <string notr="true">+7(000)000-0000</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_last_name">
          <property name="toolTip">
           <string>Введите вашу фамилию</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QRadioButton" name="radioButton_man">
          <property name="toolTip">
           <string>Выберите один из пунктов</string>
          </property>
          <property name="text">
           <string>Мужской</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_first_name">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_middle_name">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_last_name">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_user">
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_phone">
          <property name="text">
           <string>Телефон</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QDateEdit" name="dateEdit_birth_date">
          <property name="toolTip">
           <string>Введите дату вашего рождения</string>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_gender">
          <property name="text">
           <string>Пол</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_birth_date">
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_middle_name">
          <property name="toolTip">
           <string>Введите ваше отчество</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QRadioButton" name="radioButton_woman">
          <property name="toolTip">
           <string>Выберите один из пунктов</string>
          </property>
          <property name="text">
           <string>Женский</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_first_name">
          <property name="toolTip">
           <string>Введите ваше имя</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save_profile">
        <property name="text">
         <string>Сохранить профиль</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_user</tabstop>
  <tabstop>lineEdit_first_name</tabstop>
  <tabstop>lineEdit_last_name</tabstop>
  <tabstop>lineEdit_middle_name</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>radioButton_man</tabstop>
  <tabstop>radioButton_woman</tabstop>
  <tabstop>dateEdit_birth_date</tabstop>
  <tabstop>pushButton_save_profile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
